cluster/gce/gci/configure-helper_test.go

```diff
@@ -0,0 +1,215 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gci
+
+import (
+	"fmt"
+	"strings"
+
+	"testing"
+
+	"os"
+	"path/filepath"
+
+	"encoding/base64"
+	"io/ioutil"
+
+	"k8s.io/api/core/v1"
+)
+
+const (
+	// Template for defining the environment state of configure-helper.sh
+	// The environment of configure-helper.sh is initially configured via kube-env file. However, as deploy-helper
+	// executes new variables are created. ManifestTestCase does not care where a variable came from. However, future
+	// test scenarios, may require such a distinction.
+
```

Comment:
Make this a single comment block. Either remove this newline or add blank //

Reply:
Done.